setup/paths.sh

```diff
@@ -37,38 +37,27 @@ else
     source $BUILD_HOME/cmsgemos/setup/paths.sh
 fi
```

Comment:
Yes, they depend, but the point is that it should *already* be set up elsewhere, this script should simply tell the user that they have not done other things in life correctly.  The same argument is why I've requested changes in `vfatqc-python-scripts`